departure.gemspec

```
@@ -23,7 +23,7 @@ Gem::Specification.new do |spec|
 
   spec.add_runtime_dependency 'railties', '>= 6.0.0', '!= 7.0.0', '< 7.2.0'
   spec.add_runtime_dependency 'activerecord', '>= 6.0.0', '!= 7.0.0', '< 7.2.0'
-  spec.add_runtime_dependency 'mysql2', '>= 0.4.0', '<= 0.5.5'
+  spec.add_runtime_dependency 'mysql2', '>= 0.4.0', '< 1'
```

This is quite a large degree of flexibility. If we go this far, I wonder if an upper limit is useful at all.

Did you consider this?

```suggestion
  spec.add_runtime_dependency 'mysql2', '>= 0.4.0', '< 0.6.0'
```